x.c

```diff
@@ -65,6 +65,9 @@ static void ttysend(const Arg *);
 /* config.h for applying patches and the configuration. */
 #include "config.h"
 
+/* Calculate count of spare fonts */
+int fc;
```

Comment:
i think we need some better top-level variable names  in 1000+ sloc file :)

Reply:
I think we need, too, the thing is, there are multiple variables in the slock file that are as good as "fc"...  I will change however this, as it comes from the font2 patch, not from my changes.

Reply:
thanks, i think we could make new code more maintainable in the future thats always good